api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectorSpec.java

```diff
@@ -42,38 +41,29 @@ public String getClassName() {
         return className;
     }
 
+    public void setClassName(String className) {
+        this.className = className;
+    }
+
     @Description("The maximum number of tasks for the Kafka Connector")
     @Minimum(1)
     public Integer getTasksMax() {
         return tasksMax;
     }
 
-    @Description("A list of topics to use as input for the Kafka Connector")
-    public List<String> getTopics() {
-        return topics;
+    public void setTasksMax(Integer tasksMax) {
+        this.tasksMax = tasksMax;
     }
 
-    @Description("The Config for the Kafka Connector Spec.")
-    public List<KafkaConnectorConfig> getConfig() {
+    @Description("The Kafka Connector configuration. The following properties cannot be set: " + FORBIDDEN_PARAMETERS)
```

Comment:
Handling forbidden "prefixes" or "parameters" is not just about the description. I cannot find where in this PR we are really avoiding users to set them. You can take a look at how and where for example `KafkaConfiguration` class is used handling forbidden and exceptions on config parameters. Anyway I would like to know what @tombentley @scholzj think about doing in this way using forbidden before you make a change and we have no consensus :-)

Reply:
@ppatierno  I can try to find that use in the code until we decide if it's the approach we want to follow. Thanks for your inputs/feedback, really useful :)

Reply:
I'm not really convinced it's worth defining a new subclass of `AbstractConfiguration` to handle this when we have precisely 2 parameters which would be reserved and they're currently dealt with in ~3 lines of code and adding a warn log would only add another ~2 lines. Wdyt @scholzj, simplicity or consistency?

Reply:
I would be fine with it. But I do not see it being handled in the code anywhere right now.

Reply:
Tbh I am for consistency ... I don't see huge complexity even if it's just for two parameters.